plugins/Login/lang/en.json

```diff
@@ -68,6 +68,11 @@
         "BySigningUpPrivacyPolicyAndTermsAndCondition": "By signing up, I accept the %1$sprivacy policy%2$s and the %3$sterms &amp; conditions%4$s",
         "AcceptPrivacyPolicy": "You need to accept the privacy policy.",
         "AcceptTermsAndCondition": "You need to accept the terms &amp; conditions.",
-        "AcceptPrivacyPolicyAndTermsAndCondition": "You need to accept the privacy policy and the terms &amp; conditions."
+        "AcceptPrivacyPolicyAndTermsAndCondition": "You need to accept the privacy policy and the terms &amp; conditions.",
+        "CreatePassword": "Create Password",
+        "CreatePasswordDescription": "Make sure you remember this password and keep it in a safe place.",
+        "Password": "Password",
+        "ConfirmPassword": "Confirm Password",
+        "Confirm": "Confirm"
```

Comment:
'Confirm' and 'Password' translations already exist in the default lang files (Common). I'm not 100% sure whether we should put the generic 'Confirm password' and 'Create password' there as well and only keep the description key here. What do you reckon @sgiehl?

Reply:
@michalkleiner @sgiehl Moved to General translation as suggested